public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -447,3 +451,17 @@ export function buildGridItemForPanel(panel: PanelModel): SceneGridItemLike {
 }
 
 const isAdhocVariable = (v: VariableModel): v is AdHocVariableModel => v.type === 'adhoc';
+
+function registerDashboardSceneTracking(model: DashboardModel, version?: number) {
+  return () => {
+    const unsetEchoMetaExtensions = getEchoSrv().setMetaExtensions({
+      scenesView: true,
+    });
+
+    trackDashboardLoaded(model, version);
```

Comment:
Is the goal of MetaExtensions to add meta only for an execution context?

My two cents. I see a danger here to corrupt events metadata if we don't unset the meta extensions or we set meta with the same name that other meta set by lower level components. If that happens any tracking done during that execution context contains metadata that might be very specific to the product it is running. 

For example, if I add instrumentation to an enriched button, for example, the DashGPT one. This button is rendered as part of the settings view which is a scene. ANY component using tracking will get that metadata while the scene is alive. 

On the other side, I don't have any better solution on top of my head

Reply:
> ANY component using tracking will get that metadata while the scene is alive.

And that's correct as it's performed in a view that's rendering a scene and this is the execution context.

Reply:
I am failing to understand these concerns 😅, so just to gain more clarity:
 
When we set the MetaExtension,  we are adding this `"scenesView: true"` to the metadata of each event triggered inside a DashboardScene right? That will give us insights about whether user interactions are performing on the same level or maybe lower if weird things are happening. 

Am I getting it right?